### PR TITLE
Bowling challenge

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by WillDixon93** [comment]
can't get the strike and strike bojnus to work, seems to think its NaN instead of an integer.

File: src/bowling.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  constructor(){
+    this.rolls = [];
+  }
+
+  roll(pins) {
+    this.rolls.push(pins)
+  }
+
+  get score() {
```
Comment (chrismabdo) [inline src/bowling.js:11]: Good use of a get method, haven't used them myself seems to have be useful here

File: src/bowling.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    let score = 0;
+    let rollIndex = 0;
+
+    for (let frameIndex = 0; frameIndex < 10; frameIndex++) {
```
Comment (chrismabdo) [inline src/bowling.js:15]: Nice use of a loop to keep turns inclusive of rounds 1-10

File: src/bowling.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      
+      if (this.isStrike(rollIndex)) {
+        score += this.strikeBonus(rollIndex);
+        rollIndex++;
+        continue;
```
Comment (chrismabdo) [inline src/bowling.js:20]: Haven't seen the 'continue;' keyword before. Is there a reason why its needed here but not in other places?

File: spec/bowlSpec.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+   game.roll(1);
+   expect(game.score).toEqual(14)
+ });
+
+//  it('should return 300 for a perfect game', function(){
+//    rollMany(12, 10)
+//    expect(game.score).toEqual(300);
+//  })
+
+ function rollMany(pins, rolls) {
```
Comment (chrismabdo) [inline spec/bowlSpec.js:39]: Nice use of a function here, keeps tests thorough but clean

File: spec/bowlSpec.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  for (let i = 0; i < 20; i++) {
+    game.roll(pins);
+ }
+}
+});
```
Comment (chrismabdo) [inline spec/bowlSpec.js:44]: Perhaps could add in a few more tests such as consecutive strikes/consecutive spares, test against edge cases etc.
